<launch>
  <arg name="verbose" default="false" />
  <arg unless="$(arg verbose)" name="verbose_args" value="" />
  <arg     if="$(arg verbose)" name="verbose_args" value="--verbose" />

  <arg name="state_logging" default="false" />
  <arg unless="$(arg state_logging)" name="state_logging_args" value="" />
  <arg     if="$(arg state_logging)" name="state_logging_args" value="--state-logging=true" />

  <arg name="no_gui" default="false" />
  <arg unless="$(arg no_gui)" name="gui_args" value="" />
  <arg     if="$(arg no_gui)" name="gui_args" value="--no-gui" />

  <arg name="fill_demo_shipment" default="false" />
  <arg unless="$(arg fill_demo_shipment)" name="fill_demo_shipment_args" value="" />
  <arg     if="$(arg fill_demo_shipment)" name="fill_demo_shipment_args" value="--fill-demo-shipment" />

  <arg name="load_moveit" default="true" />
  <arg unless="$(arg load_moveit)" name="load_moveit_args" value="" />
  <arg if="$(arg load_moveit)" name="load_moveit_args" value="--load-moveit" />

  <arg name="load_gantry_moveit" default="false" />
  <arg unless="$(arg load_gantry_moveit)" name="load_gantry_moveit_args" value="" />
  <arg if="$(arg load_gantry_moveit)" name="load_gantry_moveit_args" value="--load-gantry-moveit" />

<arg name="load_kitting_moveit" default="false" />
  <arg unless="$(arg load_kitting_moveit)" name="load_kitting_moveit_args" value="" />
  <arg if="$(arg load_kitting_moveit)" name="load_kitting_moveit_args" value="--load-kitting-moveit" />

  <node name="ariac_sim" pkg="nist_gear" type="gear.py"
        args="$(arg verbose_args)
          $(arg state_logging_args)
          $(arg gui_args)
          $(arg load_moveit_args)
          $(arg load_gantry_moveit_args)
          $(arg load_kitting_moveit_args)
          $(arg fill_demo_shipment_args)
          --visualize-sensor-views
          -f /home/rohith/comp_config/kitting1.yaml
          $(find nist_gear)/config/user_config/sample_user_config_misc.yaml.yaml
          " required="true" output="screen" />

<!-- find ariac_2021_submission)/config/my_gear_conf.yaml -->
  <!-- <group ns='ariac/gantry'>
    <include file="$(find gantry_moveit_config)/launch/moveit_rviz.launch">
      <arg name="rviz_config" value="$(find gantry_moveit_config)/launch/moveit.rviz"/>
    </include>
  </group> -->
  <!-- <group ns='ariac/kitting'>
    <include file="$(find kitting_moveit_config)/launch/moveit_rviz.launch">
      <arg name="rviz_config" value="$(find kitting_moveit_config)/launch/moveit.rviz"/>
    </include>
  </group> -->
</launch>